### ISSUE TITLE
Finetuing the "finetuned" model on custom dataset

### ISSUE THREAD

**Issue by ajinkya-kulkarni** [Lee-Gihun/MEDIAR#18]
Hello team, thanks for this repo.
I am wondering if the "finetuned" model can be further finetuned by training it on a custom dataset.

Thanks!
Ajinkya

**Comment by Lee-Gihun** [comment]
Absolutely! It can be further fine-tuned on custom datasets for more specific customization.

However, I'd like to offer some notes for achieving better results:

Fine-tuning on custom datasets improves the model's performance on the new distribution but generally causes the model to partially forget its knowledge of the previous distribution. This extent of forgetting depends on how similar your custom data is to the data used in the pre-training or fine-tuning (challenge datasets in this context).

My first suggestion is to set a small learning rate and a small number of epochs, then observe the overall performance to assess how well the model fits your data.

**Comment by Lee-Gihun** [comment]
Let me close this issue for now. Please feel free to reopen it if there are further things to discuss!
